modules/control/proto/BUILD

```diff
@@ -18,6 +18,7 @@ proto_library(
         "leadlag_conf.proto",
         "lon_controller_conf.proto",
         "mpc_controller_conf.proto",
+        "mrac_conf.proto",
```

Comment:
BTW, this BUILD target keeps piling up which is not good.

Reply:
Will re-organize the proto_library with a new pr